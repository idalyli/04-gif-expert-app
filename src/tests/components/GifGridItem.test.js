import React from 'react'
const { shallow } = require("enzyme")
const { GifGridItem } = require("../../components/GifGridItem")
const { GifExpertApp } = require("../../GifExpertApp")

describe('Pruebas GifGridItem', () => {
   


    test('Debe mostrar <GifGridItemp correctamente /> ', () => {
        //const [categories, setCategories] = useState(['One Punch'])
        const title="Un titulo";
        const url="https://localhost/algo.com"
        const wrapper= shallow(<GifGridItem url={url} title={title} />)
        expect(wrapper).toMatchSnapshot();
    })


    
})