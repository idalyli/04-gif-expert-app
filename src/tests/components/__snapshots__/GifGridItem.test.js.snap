// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas GifGridItem Debe mostrar <GifGridItemp correctamente />  1`] = `
<div
  className="card animate__animated animate__bounce"
>
  <img
    alt="Un titulo"
    src="https://localhost/algo.com"
  />
  <p>
    Un titulo
  </p>
</div>
`;
